cli/tests/test_split_command.rs

```diff
@@ -129,14 +129,14 @@ fn test_split_by_paths() {
     test_env.set_up_fake_editor();
     let (stdout, stderr) = test_env.jj_cmd_ok(&repo_path, &["split", "-r", "@-", "nonexistent"]);
     insta::assert_snapshot!(stdout, @"");
-    insta::assert_snapshot!(stderr, @r###"
-    Warning: The given paths do not match any file: nonexistent
+    insta::assert_snapshot!(stderr, @r#"
+    Warning: No changes have been selected, the first commit will be empty
```

Comment:
The current message was added by you in #1168. I wonder if you included it as a reminder to the user. Perhaps they did `jj split -r X *.rs` and the `*.rs` gets resolved by their shell and doesn't include a file that was added in X. But maybe that's a low-enough risk that we don't need to worry about it.

Reply:
Yeah, I don't remember why I added this, it was a long time ago.